bin/webpack-helpers.js

```diff
@@ -136,8 +136,8 @@ const experimentalMainEntry = {
 
 const mainEntry =
 	process.env.WOOCOMMERCE_BLOCKS_PHASE === 'experimental'
-		? Object.assign( {}, stableMainEntry, experimentalMainEntry )
-		: Object.assign( {}, stableMainEntry );
+		? { ...stableMainEntry, ...experimentalMainEntry }
+		: { ...stableMainEntry };
```

Comment:
Does the last condition need to be a new object? Probably can just be `stableMainEntry`? The same comment applies to the `stableFrontendEntry` implementation.

Reply:
get so used to writing immutable objects  😅fixed.